shared/javascripts/options.js

```diff
@@ -364,7 +377,8 @@ if (Privly === undefined) {
     if (typeof glyph !== 'object' || glyph === null) {
       throw new Error('invalid argument');
     }
-    if (typeof glyph.color !== 'string' || typeof glyph.cells !== 'object' || glyph.cells === null || glyph.cells.constructor !== Array) {
+    if (typeof glyph.color !== 'string' || typeof glyph.cells !== 'object' || 
+        glyph.cells === null || Object.prototype.toString.call(glyph.cells) !== "[object Array]") {
```

Comment:
`glyph.cells.constructor !== Array` will not work for CommonJS modules because the `Array` global object isn't the same one in two different modules/env.